### ISSUE TITLE
Comments in .csslintrc

### ISSUE THREAD

**Issue by smonusbonus** [jgable/grunt-lesslint#65]
Hey there,

it would actually be really cool if you could have comments in the .csslintrc file for options that you're not using. Right now it throws an error when trying to parse json which includes slashes (see screenshot).

![screen shot 2015-04-23 at 12 13 15](https://cloud.githubusercontent.com/assets/1876194/7294992/685aa204-e9b2-11e4-8b10-933b3f4c06b6.png)

Cheers

**Comment by davidparsson** [comment]
The JSON format does not support comments, but you could write comments in the JSON data, like this:

```JSON
{
  "#": "Allow IDs in selectors",
  "ids": false,
  "#": "Allow our special properties",
  "known-properties": false
}
```

**Comment by davidparsson** [comment]
Perhaps it makes sense to support this anyway, since [grunt-contrib-csslint does](https://github.com/gruntjs/grunt-contrib-csslint#csslintrc).

**Comment by davidparsson** [comment]
Fixed in grunt-lesslint@1.4.0.
